tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -21,6 +21,8 @@ x) BUILD_EXTR_PAR=${OPTARG};;
 c) CUDA_VER=${OPTARG};;
 # x86 or other, only for ubuntu16.04 os
 a) BUILD_ARCH=${OPTARG};;
+# openvino version tag: 2018_R5, 2019_R1 (Default is 2018_R5)
+v) OPENVINO_VERSION=${OPTARG};; 
```

Comment:
in practice, will this option really be used? 
will we support building multiple versions of openvino from source? or will it always be the latest stable version?

Reply:
This in the near future should default to the latest stable OpenVINO version

Reply:
Agreed. Support for the latest versions will be coming out in the near future.